pkg/bff/api/v1/api.go

```diff
@@ -43,8 +43,9 @@ type BFFClient interface {
 	MemberDetails(context.Context, *MemberDetailsParams) (*MemberDetailsReply, error)
 
 	// Registration form
-	LoadRegistrationForm(context.Context, *LoadRegistrationFormParams) (*RegistrationForm, error)
+	LoadRegistrationForm(context.Context, *RegistrationFormParams) (*RegistrationForm, error)
 	SaveRegistrationForm(context.Context, *RegistrationForm) (*RegistrationForm, error)
+	DeleteRegistrationForm(context.Context, *RegistrationFormParams) (*RegistrationForm, error)
```

Comment:
Maybe we should call it `ClearRegistrationForm`?

Reply:
That probably fits better

Reply:
What about `ResetRegistrationForm`?

Reply:
That works for me too.